library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -81,6 +89,27 @@ data class Conversations(
         )
     }
 
+    fun newGroup(accountAddresses: List<String>): Group {
+        val group = runBlocking {
+            libXMTPConversations?.createGroup(accountAddresses)
+                ?: throw XMTPException("Client does not support Groups")
+        }
+        return Group(client, group)
+    }
+
+    suspend fun syncGroups() {
+        libXMTPConversations?.sync()
+    }
+
+    fun listGroups(): List<Group> {
+        return runBlocking {
+            syncGroups()
```

Comment:
Same here. Probably don't want to run sync for them?

Reply:
It might depend on whether we want to allow people to page through the `listGroups` results. If you're paging, we definitely don't want to re-check on each page. If we're always returning the entire list (setting null on the options below), maybe it's fine to do the check implicitly?